lib/ts/config.ts

```diff
@@ -202,6 +202,32 @@ export async function getFrontendOptions({ manager }: UserFlags): Promise<Questi
                 setup: [`${manager} install`],
             },
         },
```

Comment:
We also need to modify the index.ts near line number 131 to say that Nuxt.js app doesn't support multi tenancy

Reply:
Done